external/sql/storm-sql-core/src/jvm/org/apache/storm/sql/compiler/ExprCompiler.java

```diff
@@ -67,8 +68,9 @@ public ExprCompiler(PrintWriter pw, JavaTypeFactory typeFactory) {
   public String visitInputRef(RexInputRef rexInputRef) {
     String name = reserveName();
     String typeName = javaTypeName(rexInputRef);
+    String boxedTypeName = boxedJavaTypeName(rexInputRef);
```

Comment:
Actually converting Object to primitive type is allowed for Java 7 or upper, but Janino throws error while compiling script so I just converted to boxed type.